.travis.yml

```diff
@@ -0,0 +1,41 @@
+sudo: required
+language: python
+services:
+  - docker
+env:
+    DOCKER_COMPOSE_VERSION: 1.10.1
+before_install:
+- sudo apt-add-repository 'deb http://archive.ubuntu.com/ubuntu trusty-backports universe'
+- sudo apt-get update -qq
+- docker-compose --version
+- sudo apt-get update
+- sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes -y docker-ce
+- sudo rm /usr/local/bin/docker-compose
+- curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+- chmod +x docker-compose
+- sudo mv docker-compose /usr/local/bin
+- docker-compose --version
+before_script:
+- export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
+- git checkout $BRANCH
+- echo "" >> .prod.env
+- echo "DJANGO_SECRET_KEY=${DJANGO_SECRET_KEY}" >> .prod.env
+- echo "" >> .prod.env
+- echo "DATABASE_URL=mysql://${DATABASE_USER}:${DATABASE_PASSWORD}@${DATABASE_HOST}/${DATABASE_NAME}" >> .prod.env
+- echo "" >> .prod.env
+- echo "IMPACT_API_V0_SECURITY_KEY=${IMPACT_API_V0_SECURITY_KEY}" >> .prod.env
+- echo "" >> .prod.env
+- echo "IMPACT_API_V0_IMAGE_PASSWORD=${IMPACT_API_V0_IMAGE_PASSWORD}" >> .prod.env
+- echo "" >> .prod.env
```

Comment:
I really don't like the repetition here. probably can be rewritten in a bash loop (except for the DATABASE_URL).